xalia/Win32/AccessibleProvider.cs

```diff
@@ -676,7 +677,7 @@ private async Task PollChildren()
                             break;
                     }
                     return result;
-                }, Tid);
+                }, CommandThreadPriority.Query);

```

Comment:
Was this supposed to be +1 originally and you've fixed it here, or should this be ` CommandThreadPriority.User` like the other `Tid` cases?